src/components/TodosList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+const TodosList = (props) => {
+  const { todos, checkHandler, deleteHandler } = props;
+  return (
+    <ul>
+      {todos.map((todo) => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          checkHandler={checkHandler}
+          deleteHandler={deleteHandler}
+        />
+      ))}
+    </ul>
+  );
+};
+
+TodosList.propTypes = {
+  todos: PropTypes.arrayOf(Object).isRequired,
```

Comment:
Nice way of solving the 'array not accepted by linters'.

Reply:
It's a proposed solution by the linters. It makes sense to be specific